tools/deps_info.py

```diff
@@ -167,7 +166,6 @@
   'gmtime_r':  ['malloc'],
   'localtime': ['_get_tzname', '_get_daylight', '_get_timezone', 'malloc'],
   'localtime_r': ['_get_tzname', '_get_daylight', '_get_timezone', 'malloc'],
-  'longjmp': ['setThrew'],
```

Comment:
I wonder thy this method of selectively including setThrew only when needed stopped working?   It seems like it should still work.

Since SUPPORT_LONGJMP is enabled by default it seems shame to unconditionally pull in `setThrew` since that will increate the size of all programs that don't use setjmp.

Reply:
This file is for JS -> C dependencies, and now `longjmp` doesn't even exist and `emscripten_longjmp` is in the native code, this doesn't seem to be necessary anymore. Is this not correct? 

If `setThrew` is always pulled in, `emscripten_longjmp` will be pulled in too. Now they are just normal native library functions. Are unused library functions removed?